build.rs

```diff
@@ -161,7 +168,9 @@ where
 {
     let link = if is_dynamic() { "dylib" } else { "static" };
     let path = path.as_ref();
-    copy(path, LIBRARY_PATH.as_path())?;
+    // Some `git` files are created with write protection, so we must allow the recursive
+    // copy to clobber files that are read-only.
+    copy(path, LIBRARY_PATH.as_path(), true)?;
```

Comment:
Minor nitpick, but I'm usually always against `bool` arguments, as you always have to go to the function definition to see what `true` or `false` mean.

Reply:
Good point, agreed.

I'm not sure how to solve this in a general sense on a single-file script like this though. With some encapsulation, I typically write an "impl" style function in an inner module, and wrap it with appropriately variants covering every `bool` possibility. Is it worth it doing that on a `build.rs` script you think? Alternatively I can have an enum for the argument, which may be simpler here.

Reply:
Does the `copy` function actually need to be marked `pub`? If not, then just defining a simple `enum` in `build.rs` seems pretty straightforward unless I'm missing something.

Reply:
I'll go with the `enum` choice then :)

Reply:
Thanks!